### PR TITLE
XCM docs migration

### PR DESCRIPTION
**Pull request by DrW3RK**

### CODE REVIEW

File: polkadot-wiki/sidebars.js [hunk 1]
```diff
@@ -370,6 +370,145 @@ module.exports = {
...
+                "learn/xcm/journey/locks/learn-xcm-docs-journey-locks",
+                "learn/xcm/journey/learn-xcm-docs-journey-channels",
+              ],
+            },
+            "learn/xcm/executor_config/learn-xcm-docs-config-deep-dive",
+            "learn/xcm/learn-xcm-docs-testing",
+            {
+              type: "category",
+              label: "Reference",
+              description: "References.",
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]:
```suggestion
              description: "XCVM Registers and Glossary",
```

File: polkadot-wiki/sidebars.js [hunk 2]
```diff
@@ -370,6 +370,145 @@ module.exports = {
...
+                "learn/xcm/journey/learn-xcm-docs-journey-version",
+                "learn/xcm/journey/locks/learn-xcm-docs-journey-locks",
+                "learn/xcm/journey/learn-xcm-docs-journey-channels",
+              ],
+            },
+            "learn/xcm/executor_config/learn-xcm-docs-config-deep-dive",
+            "learn/xcm/learn-xcm-docs-testing",
+            {
+              type: "category",
+              label: "Reference",
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]:
```suggestion
              label: "References",
```

File: polkadot-wiki/sidebars.js [hunk 3]
```diff
@@ -370,6 +370,145 @@ module.exports = {
...
+              description: "References.",
+              link: {
+                type: 'generated-index',
+                title: "Reference",
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]:
```suggestion
                title: "References",
```

File: polkadot-wiki/sidebars.js [hunk 4]
```diff
@@ -370,6 +370,145 @@ module.exports = {
...
+                description: "References.",
+                slug: '/learn/xcm/reference',
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]:
```suggestion
                slug: '/learn/xcm/references',
```

File: polkadot-wiki/sidebars.js [hunk 5]
```diff
@@ -370,6 +370,145 @@ module.exports = {
...
+                },
+                "learn/xcm/fundamentals/learn-xcm-docs-fundamentals-multiasset",
+                "learn/xcm/fundamentals/learn-xcm-docs-fundamentals-xcvm",
+                "learn/xcm/fundamentals/learn-xcm-docs-fundamentals-fees",
+              ],
+            },
+            {
+              type: "category",
+              label: "A Journey through XCM",
+              description: "A Journey through XCM.",
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]:
```suggestion
              description: "A practical introduction to the XCM features",
```

File: polkadot-wiki/sidebars.js [hunk 6]
```diff
@@ -370,6 +370,145 @@ module.exports = {
...
+              link: {
+                type: 'generated-index',
+                title: "A Journey through XCM",
+                description: "A Journey through XCM.",
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]:
```suggestion
                description: "A practical introduction to the XCM features.",
```

**Review by DrW3RK — COMMENTED** [review]
As this would be the landing page https://staging.polkadot.network/docs/learn/xcm try to add descriptions that better summarize the sections. I tried adding a few.

**Review by filippoweb3 — APPROVED** [review]

**Pull request merged**